src/actions/common.js

```diff
@@ -195,13 +194,6 @@ export const loadInitialData = clientId => (dispatch, getState) => {
       type: types.SET_INITIAL_DATA_LOADING,
       payload: false
     })
-
-    // On initial load route has changed, but data was not loaded,
```

Comment:
What about the use-case described in the comment? Is this not the case anymore?

Reply:
I can't reproduce them anymore:(

Reply:
This part was added during a big rewrite! We can't reproduce it nor is there and previous issue. Let's merge it and see if we run into troubles during the test phase.